src/qt/configurenamedialog.cpp

```diff
@@ -98,4 +111,13 @@ void ConfigureNameDialog::onDataEdited(const QString &name)
 {
     ui->dataSize->setText(tr("%1 / %2").arg(name.size()).arg(MAX_VALUE_LENGTH_UI));
     ui->dataSize->resize(ui->dataSize->fontMetrics().horizontalAdvance(ui->dataSize->text()), ui->dataSize->height());
+
+    std::string data = ui->dataEdit->text().toStdString();
+
+    if(IsValidJSON(data)){
+        ui->labelValidJSON->setText(tr("Valid JSON text."));
+    }else{
+        ui->labelValidJSON->setText(tr("JSON data inputted is invalid or not minimal."));
```

Comment:
The "minimal" check is being saved for the next PR, right?

Reply:
Yeah, that is correct